sys-apps/onerng/onerng-3.6.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
+
+inherit distutils-r1 udev
+
+DESCRIPTION="Software for the Open Hardware Random Number Generator called OneRNG"
+HOMEPAGE="http://www.onerng.info/"
+SRC_URI="https://github.com/OneRNG/onerng.github.io/raw/master/sw/${PN}_${PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+RDEPEND="app-crypt/gnupg
+	dev-python/python-gnupg[${PYTHON_USEDEP}]
+	sys-apps/rng-tools
+	sys-process/at
+	virtual/udev"
+
+DEPEND="virtual/pkgconfig
+	virtual/udev"
+
+S="${WORKDIR}/${PN}_${PV}"
+
+src_configure() {
+	# Run configure
+	econf
+}
+
+src_compile() {
+	# Run make
+	emake
+}
+
+src_install() {
+	# Install files
+	emake DESTDIR="${D}" install
+
+	# Install docs
+	local DOCS=( "AUTHORS" "ChangeLog" "INSTALL" "NEWS" "README" )
```

Comment:
why not just default `einstalldocs` ?

Reply:
In that case, it only installs AUTHORS, ChangeLog and README. I wanted also to have INSTALL and NEWS for additional information.